src/Api/Search.php

```diff
@@ -290,22 +290,23 @@ public static function search($params)
                 );
             }
 
-            if (isset($params['from']) && !empty($params['from'])
-                && isset($params['to']) && !empty($params['to'])) {
-                $from = date('Y-m-d', strtotime($params['from']));
-                $to = date('Y-m-d', strtotime($params['to']));
+            $timeZoneString = wp_timezone_string();
+            $today = new \DateTime("now", new \DateTimeZone($timeZoneString));
+            $timezoneOffset = $today->format('P');
 
-                $timeZoneString = wp_timezone_string();
-                $today = new \DateTime("now", new \DateTimeZone($timeZoneString));
-                $timezoneOffset = $today->format('P');
+            if (isset($params['from']) && !empty($params['from'])) {
+                $from = date('Y-m-d', strtotime($params['from']));
                 $from .= "T00:00:00" . $timezoneOffset;
-                $to .= "T23:59:59" . $timezoneOffset;
-
                 $args['filters'][] = array(
                     'field' => 'start',
                     'operation' => 'ge',
                     'value' => $from,
                 );
+            }
+
+            if (isset($params['to']) && !empty($params['to'])) {
+                $to = date('Y-m-d', strtotime($params['to']));
```

Comment:
Im confused a bit, because the `ADMWPP_SEARCH_DATE_FORMAT` says the format is `mm-dd-yy` but here we are using a `Y-m-d hh:mm:ss Timezone` format?